server/tserver/src/main/java/org/apache/accumulo/tserver/TabletIteratorEnvironment.java

```diff
@@ -122,6 +127,11 @@ public boolean isFullMajorCompaction() {
     return fullMajorCompaction;
   }
 
+  @Override
+  public boolean isUserCompaction() {
+    return userCompaction;
```

Comment:
A sanity check like the one in `isFullMajorCompaction()` that checks if `scope` is `majc` would be nice.